### PR TITLE
Setup Oppiabot stale checks for Oppia Android

### PR DESCRIPTION
**Pull request by jameesjohn**
## Explanation
This PR sets up oppiabot stale checks.
It will mark PRs without updates for 6 days as stale, and close them after 4 days.

## Checklist

- [ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [ ] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [ ] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by jameesjohn** [comment]
PTAL @BenHenning

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 7
+
+# Number of days of inactivity before a stale Issue or Pull Request is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 3
```
Comment (BenHenning) [inline .github/stale.yml]: Could we use 7, instead, for parity with the days until stale? 7 is a bit long for closing, but 3 seems to short so I'm leaning toward the longer range here.

**Review by BenHenning — CHANGES_REQUESTED** [review]
Thanks @jameesjohn! Just had one comment--PTAL.

File: .github/stale.yml (hunk 1 above)
Comment (jameesjohn) [reply .github/stale.yml]: Days until close being 3 here means it will be 3 days after being stale ie 10 days in total.

**Review by jameesjohn — COMMENTED** [review]
Hi, @BenHenning I responded to your comment, PTAL.
Also, how do I fix the failing test? It seems like a code owners check.

File: .github/stale.yml (hunk 1 above)
Comment (BenHenning) [reply .github/stale.yml]: I actually meant that we should use 14 days in total @jameesjohn. My concern is that contributors only have 3 days to follow up after the initial comment that the PR is stale to provide changes before the PR is closed.

**Comment by BenHenning** [comment]
@jameesjohn apologies for not following up earlier--I tend to only look at assigned PRs since those require my active attention.

To fix the codeowners check you'll need to add the new file to CODEOWNERS (making me the codeowner here should be fine).

File: .github/stale.yml (hunk 1 above)
Comment (jameesjohn) [reply .github/stale.yml]: Done.

**Comment by jameesjohn** [comment]
Done. 
PTAL @BenHenning. Thanks

**Review by BenHenning — APPROVED** [review]
Thanks @jameesjohn! This LGTM--I appreciate you making this config change for us.

**Review by seanlip — APPROVED** [review]

**Pull request merged**